src/main/java/org/elasticsearch/index/query/SimpleQueryStringBuilder.java

```diff
@@ -203,4 +273,164 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
     protected String parserName() {
         return SimpleQueryStringParser.NAME;
     }
+
+    /**
+     * Class encapsulating the settings for the SimpleQueryString query, with
+     * their default values
+     */
+    public static class Settings {
+        private Locale locale = Locale.ROOT;
+        private boolean lowercaseExpandedTerms = true;
+        private boolean lenient = false;
+        private boolean analyzeWildcard = false;
+
+        public Settings() {
+
+        }
+
+        public Locale locale() {
+            return this.locale;
+        }
+
+        public Settings locale(Locale locale) {
+            Preconditions.checkNotNull(locale);
+            this.locale = locale;
+            return this;
+        }
+
+        public boolean lowercaseExpandedTerms() {
+            return this.lowercaseExpandedTerms;
+        }
+        public Settings lowercaseExpandedTerms(boolean lowercaseExpandedTerms) {
+            this.lowercaseExpandedTerms = lowercaseExpandedTerms;
+            return this;
+        }
+
+        public boolean lenient() {
+            return this.lenient;
+        }
+        public Settings lenient(boolean lenient) {
+            this.lenient = lenient;
+            return this;
+        }
+
+        public boolean analyzeWildcard() {
+            return analyzeWildcard;
+        }
+
+        public Settings analyzeWildcard(boolean analyzeWildcard) {
+            this.analyzeWildcard = analyzeWildcard;
+            return this;
+        }
+        
+        @Override
+        public int hashCode() {
+            return Objects.hash(locale, lowercaseExpandedTerms, lenient, analyzeWildcard);
+        }
+        
+        @Override
+        public boolean equals(Object obj) {
+            if (this == obj) {
+                return true;
+            }
+            if (obj == null || getClass() != obj.getClass()) {
+                return false;
+            }
+            Settings other = (Settings) obj;
+            
+            // checking the return value of toString() for locales only. For further reasoning see
+            //  https://issues.apache.org/jira/browse/LUCENE-4021
+            return Objects.equals(locale.toLanguageTag(), other.locale.toLanguageTag()) &&
+                   (lowercaseExpandedTerms == other.lowercaseExpandedTerms) &&
+                   (lenient == other.lenient) &&
+                   (analyzeWildcard == other.analyzeWildcard);    
+        }
+    }
+
+    @Override
+    public SimpleQueryStringBuilder readFrom(StreamInput in) throws IOException {
+        SimpleQueryStringBuilder qb = new SimpleQueryStringBuilder(in.readOptionalString());
+
+        int size = in.readInt();
+        for (int i = 0; i < size; i++) {
+            String field = in.readString();
+            Float weight = in.readFloat();
+            qb.field(field, weight);
+        }
+
+        qb.flags(in.readInt());
+        qb.analyzer(in.readOptionalString());
```

Comment:
similarly to what previously discuss with Christop about whether we should use setters or set fields directly in readFrom, I think we went more for the latter when possible, I would adapt this for consistency too then